tests/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "humm-widget-cards-online-tests",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "npx cypress run",
+    "open": "npx cypress open",
+    "record": "npx cypress run --record --key 3d96ba1c-7211-4e98-bd5c-30dc82ed7d30"
+  },
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "cypress": "^8.4.0",
+    "typescript": "^4.4.3"
```

Comment:
can we use fixed versions of typescript and cypress by removing the ^?

Reply:
This should hopefully not be an issue now that i've moved my tests to the root :) cypress was added as a dependency to your package.json now